sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/CustomValidationPolicy.java

```diff
@@ -42,16 +41,12 @@ public Mono<HttpResponse> process(
                 .map(CustomValidationPolicy::ungzip);
         asyncBytes.subscribe(
             value -> {
-                ObjectMapper objectMapper = createObjectMapper();
-                try (MappingIterator<TelemetryItem> i =
-                         objectMapper.readerFor(TelemetryItem.class).readValues(value)) {
-                    while (i.hasNext()) {
-                        actualTelemetryItems.add(i.next());
-                    }
-                    countDown.countDown();
+                try {
+                    actualTelemetryItems.addAll(deserialize(value.getBytes(StandardCharsets.UTF_8)));
                 } catch (Exception e) {
                     throw new RuntimeException(e);
                 }
```

Comment:
it looks like the try/catch/throw isn't needed?

Reply:
deserialize throws exception. intellij is not happy